R/prepare_weo_2024_hybrid_in_ev_scenario.R

```diff
@@ -147,7 +157,7 @@ weo_2024_extract_power <- function(weo_2024_ext_data_regions_raw,
     weo_2024_extended_regions %>%
     dplyr::filter(
       # assumption: prior to and inclusive 2022, NZE is consistent with STEPS
```

Comment:
same as above: If we make this dependent on the year of the publication, the comment should probably be updated to reflect that